schema/db_migration.sql

```diff
@@ -0,0 +1,212 @@
+-- ids:
+--  id: bigserial
+--  idx: bytea # 16 bytes, indexed, used for lookup
+--  remainder: bytea # remainder of binary data
+--  type: smallint # 0=eoa 1=contract 2=txhash 3=blockhash
+-- TAMCHAU:
+--   "type" is postgres internal key, should use other name -> "id_type".
+--   Should ENUM instead of smallint for better performance (don't need to convert int to value later).
+--   Consider to add "created_at" column (maybe index for it also) in this table to track when a new
+--     record is created.
+--   Consider to add unique (might be partial) index on remainder to make sure no duplication txs,
+--      at least for txhash, contract...
+CREATE TYPE id_type AS ENUM ('eoa', 'contract', 'txhash', 'blockhash');
+CREATE TABLE ids (
+   id bigserial PRIMARY KEY, -- idx index created automatically for primary key (ids_pkey)
```

Comment:
please recheck the requirement, it might sound confuse at first but the logic is something like

- Input: some arbitrary length bytea `long_bytea`
- DB only support index on limited length text/bytea
- so the trick is `long_bytea` = `idx` + `remainder`
- with creating INDEX on `idx`, we can achieve both purpose: fast lookup and arbitrary length storage, hence can support wide range of application (user input arbitrary length)